libsweep/src/unix/serial.cc

```diff
@@ -26,6 +26,11 @@ static speed_t get_baud(int32_t bitrate) {
     throw error{"Only baud rate 115200 is supported at this time."};
     return -1;
   }
+
+// translate human readable bitrate to termios bitrate
+#ifdef B115200
+  return B115200;
+#endif
```

Comment:
This is a good quick-fix but we should refactor the function (or how we allow for bitrates here).

We take a bitrate argument but in case we hit the branch here the argument is never used.